packages/core/content-manager/server/services/utils/populate.js

```diff
@@ -10,7 +10,7 @@ const { PUBLISHED_AT_ATTRIBUTE } = strapiUtils.contentTypes.constants;
 const getDeepPopulate = (
   uid,
   populate,
-  { onlyMany = false, countMany = false, maxLevel = Infinity } = {},
+  { countMany = false, countOne = false, maxLevel = Infinity } = {},
```

Comment:
Because for the findMany (the list view) we count only the xToMany relations as we don't display them and we populate the xToOne relations normally (the mainField is displayed for them). So we need to be able to have a different param for each. Does that make sense?